frontend/src/components/MakerForm/MakerForm.tsx

```diff
@@ -329,6 +343,7 @@ const MakerForm = ({
       ? parseFloat(maker.amount)
       : parseFloat(Number(limits.list[index].max_amount * 0.75).toPrecision(2));
 
+    setAmountRangeEnabled(true);
```

Comment:
if `amountRangeEnabled` is true by default, this one is not needed.

Reply:
Resolved in the last commit 375eb495016decfc2bffd61172e8ab50fce5dc8a.